project/frontend/src/components/ResultsMap.js

```diff
@@ -1,12 +1,55 @@
-import React from 'react';
+import React, { Fragment, useState } from 'react';
 import GoogleMapReact from 'google-map-react';
 import Lunch from '../assets/lunch.svg';
 
 function MapContainer(props) {
   const restaurants = props.restaurants;
   const userCenter = props.userLocation;
+  const [activeMarker, setActiveMarker] = useState({});
+  const [showInfoWindows, setShowInfoWindows] = useState([
+    false,
+    false,
+    false,
+    false,
+  ]);
 
-  const MarkerIcon = () => <img src={Lunch} alt='Lunch icon' />;
+  // TODO: Add a marker for "you are here location".
+  const MarkerIcon = (props) => {
+    const infoNum = props.id;
+    console.log(showInfoWindows[infoNum]);
+    return (
+      <Fragment>
+        <img src={Lunch} alt={'lunch icon'} />
+        {showInfoWindows[infoNum] && <InfoWindow marker={activeMarker} />}
+      </Fragment>
+    );
+  };
+
+  const InfoWindow = (props) => {
+    // TODO: Remove hard coded data and add name and percent match from the backend.
+    const restaurantName = 'Applebees';
+    const percentMatch = '96%';
+    const infoWindowStyle = {
+      position: 'relative',
+      bottom: 80,
+      left: '-45px',
+      width: 220,
+      backgroundColor: 'white',
+      boxShadow: '0 2px 7px 1px rgba(0, 0, 0, 0.3)',
+      padding: 10,
+      fontSize: 14,
+      zIndex: 100,
+    };
```

Comment:
for some reason i put that im comparing this branch to my addInfoWindows branch which has this code but it still says that this is new to this branch ? confusion

Reply:
Did you branch from the addInfoWindows branch to create this branch?

Reply:
I did!